packages/theme/stories/introduction.stories.mdx

```diff
@@ -41,100 +37,106 @@ A simple example will help to illustrate some of the key concepts:
-| Navigable           | Components that change data sets or view contexts                                 | `--uitk-navigable-cursor-hover`              | `pointer`                                          |
-| Overlayable         | Components that form hierarchies in layout                                        | `--uitk-overlayable-shadow-scroll`           | `--uitk-shadow-1`                                  |
-| Selectable          | Components that have a selected state                                             | `--uitk-selectable-cursor-hover`             | `pointer`                                          |
-| Separable           | Components containing elements which should be separated into different divisions | `--uitk-separable-primary-borderColor`       | `--uitk-palette-neutral-primary-border`            |
-| Status              | Components indicating an informational, warning, error or success state           | `--uitk-status-info-borderColor`             | `--uitk-palette-info-border`                       |
-| Taggable            | Components which allow data to be organized                                       | `--uitk-taggable-background`                 | `--uitk-palette-interact-primary-background`       |
-| Text                | All text components or HTML elements                                              | `--uitk-text-primary-foreground`             | `--uitk-palette-neutral-primary-foreground`        |
-
-#### Token naming conventions
-
-Comparing the tokens above, you may have noticed the following conventions we have applied to token naming:
-
-- CSS variables in the UI Toolkit theme are prefixed `--uitk-`. This is a rule with no exceptions.
+## CSS levels
+
+Salt styles are defined at two different levels, which operate together at runtime:
+
+- **Theme styles** - consisting of CSS Variables \*
+- **Component styles** - each component contributes its own stylesheet. Many of the CSS declarations reference CSS variables defined in the theme i.e. the variables will be defined at the theme level and used at the component level.
+
+\* Also includes one _utility_ class
+
+## Theme styles
+
+The Salt theme is a collection of design tokens split into a three-tier hierarchy. The token tiers are:
+
+1. **Foundation** - Foundation tokens are the primitive values for attributes like color, size, shadow and opacity.
+2. **Palette** - semantically grouped foundational colors–neutral, navigate, interact –which allow a consistent look-and-feel when used by characteristics, rather than referencing the foundational tokens directly. For example, actionable, selectable, and editable characteristic tokens all use the 'interact' palette. If the theme needed to change the background color of interactive elements it would only have to be updated in one place, the palette, instead of within each characteristic.
```

Comment:
Numbering should be used for sequential steps, these should be bullets. Also there seems to be a bit of a mix-up at the start of the Palette description, maybe a copy and paste error?

Reply:
Thanks Sion, I've updated the copy.